db/migrate/20141215175503_rename_via_script.rb

```diff
@@ -0,0 +1,285 @@
+    add_index 'sentences_symbols', ['sentence_id', 'symbol_id'], unique: true, name: 'index_sentences_symbols_on_sentence_id_and_symbol_id'
+    remove_index 'symbol_groups', name: 'index_entity_groups_on_ontology_id_and_id'
+    add_index 'symbol_groups', ['ontology_id', 'id'], unique: true, name: 'index_symbol_groups_on_ontology_id_and_id'
+    remove_index 'symbol_mappings', name: 'index_entity_mappings_on_source_id'
+    add_index 'symbol_mappings', ['source_id'], unique: false, name: 'index_symbol_mappings_on_source_id'
+    remove_index 'symbol_mappings', name: 'index_entity_mappings_on_target_id'
+    add_index 'symbol_mappings', ['target_id'], unique: false, name: 'index_symbol_mappings_on_target_id'
+    remove_index 'mapping_versions', name: 'index_link_versions_on_link_id'
+    add_index 'mapping_versions', ['mapping_id'], unique: false, name: 'index_mapping_versions_on_mapping_id'
+    remove_index 'mapping_versions', name: 'index_link_versions_on_source_id'
+    add_index 'mapping_versions', ['source_id'], unique: false, name: 'index_mapping_versions_on_source_id'
+    remove_index 'mapping_versions', name: 'index_link_versions_on_target_id'
+    add_index 'mapping_versions', ['target_id'], unique: false, name: 'index_mapping_versions_on_target_id'
+    remove_index 'mappings', name: 'index_links_on_link_version_id'
+    add_index 'mappings', ['mapping_version_id'], unique: false, name: 'index_mappings_on_mapping_version_id'
+    remove_index 'mappings', name: 'index_links_on_ontology_id'
+    add_index 'mappings', ['ontology_id'], unique: false, name: 'index_mappings_on_ontology_id'
+    remove_index 'mappings', name: 'index_links_on_source_id'
+    add_index 'mappings', ['source_id'], unique: false, name: 'index_mappings_on_source_id'
+    remove_index 'mappings', name: 'index_links_on_target_id'
+    add_index 'mappings', ['target_id'], unique: false, name: 'index_mappings_on_target_id'
+    remove_index 'structured_proof_parts', name: 'index_structured_proof_parts_on_link_version_id'
+    add_index 'structured_proof_parts', ['mapping_version_id'], unique: false, name: 'index_structured_proof_parts_on_mapping_version_id'
+    execute "ALTER TABLE ONLY symbols DROP CONSTRAINT entities_ontology_id_fk;"
+    execute "ALTER TABLE ONLY symbols ADD CONSTRAINT symbols_ontology_id_fk FOREIGN KEY (ontology_id) REFERENCES ontologies(id) ON DELETE CASCADE;"
+    execute "ALTER TABLE ONLY oops_responses_symbols DROP CONSTRAINT entities_oops_responses_entity_id_fk;"
+    execute "ALTER TABLE ONLY oops_responses_symbols ADD CONSTRAINT oops_responses_symbols_symbol_id_fk FOREIGN KEY (symbol_id) REFERENCES symbols(id);"
+    execute "ALTER TABLE ONLY oops_responses_symbols DROP CONSTRAINT entities_oops_responses_oops_response_id_fk;"
+    execute "ALTER TABLE ONLY oops_responses_symbols ADD CONSTRAINT oops_responses_symbols_oops_response_id_fk FOREIGN KEY (oops_response_id) REFERENCES oops_responses(id) ON DELETE CASCADE;"
```

Comment:
Metrics/LineLength: Line is too long. [191/80]<br>Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.